src/xdocs/config.xml

```diff
@@ -244,6 +244,11 @@
 &lt;/module&gt;
       </source>
 
+      <p>
+        ATTENTION: default value is applied to module property value if at least one expansion
+        property is not defined.
```

Comment:
I assume this is referring to property values with multiple expansions in a single property?

This needs to be expanded. In addition why don't we add an example?

Reply:
Not only multiple expansion, any .

Reply:
I thought about example, but it is not easy to explain and might be too technical, and looks like nobody dealing with this, so we found this nuance only during pitest.

I think we don't need to make our doc to be too much extended and hard to read.